pkg/services/featuremgmt/registry.go

```diff
@@ -142,6 +142,30 @@ var (
 			Owner:        grafanaDatavizSquad,
 			Created:      time.Date(2023, time.December, 11, 7, 0, 0, 0, time.UTC),
 		},
+		{
+			Name:         "autoMigrateTablePanel",
+			Description:  "Migrate old table panel to supported table panel - broken out from autoMigrateOldPanels to enable granular tracking",
+			Stage:        FeatureStagePublicPreview,
+			FrontendOnly: true,
+			Owner:        grafanaDatavizSquad,
+			Created:      time.Date(2024, time.February, 12, 0, 0, 0, 0, time.UTC),
```

Comment:
^^^ FYI, you will need to remove the Created field when trying to merge main.  That now happens automatically https://github.com/grafana/grafana/pull/82131